lib/hyperglot/data/cbi.yaml

```diff
@@ -2,15 +2,17 @@ contributors:
 - Sergio Martins
```

Comment:
Please add yourself as contributor for this file. Other than that looks good to merge, thanks for the correction :)